packages/table/src/table.tsx

```diff
@@ -1826,6 +1826,11 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
 
         // change selection to match new focus cell location
         const newSelectionRegions = [Regions.cell(newFocusedCell.row, newFocusedCell.col)];
+        const { selectedRegionTransform } = this.props;
+        if (selectedRegionTransform != null) {
+            // tslint:disable-next-line no-unnecessary-callback-wrapper
+            newSelectionRegions.forEach(region => selectedRegionTransform(region, undefined));
```

Comment:
why the explicit `undefined`? i'm not familiar with this code.

Reply:
`selectedRegionTransform()` is a function that takes a region and a mouse event that triggered the selection. In this case, there is no mouse event so the best I could do is to pass `undefined` as the second argument. (I did not want to mess around with the typing of `selectedRegionTransform` as that would be an API breakage). Actually, I was surprised to see that TypeScript accepts `undefined` for an argument that is typed as a `MouseEvent`.

Also, strangely enough, TypeScript complains if I omit the second argument ("Expected 2-3 arguments, got 1"), so that's why we need an explicit `undefined` here.

Reply:
yeah this is not valid. it only accepts `undefined` because we have not enabled 
`strictNullChecks` compiler options.